hs_core/models.py

```diff
@@ -718,26 +729,25 @@ def update(cls, element_id, **kwargs):
         metadata_obj = kwargs['content_object']
         # get matching resource
         resource = BaseResource.objects.filter(object_id=metadata_obj.id).first()
-
+        publisher_CUAHSI = "Consortium of Universities for the Advancement of Hydrologic Science, Inc. (CUAHSI)"
```

Comment:
@hyi - yes, this is correct.  For resources where the content files are stored and published in HydroShare, the user should not be able to modify the publisher element, and it is final.